homeassistant/components/binary_sensor/__init__.py

```diff
@@ -37,6 +37,12 @@ class BinarySensorDeviceClass(StrEnum):
     # On means charging, Off means not charging
     BATTERY_CHARGING = "battery_charging"
 
+    # On means carbon monoxide detected, Off means no carbon monoxide (clear)
+    CO = "carbon_monoxide"
+
+    # On means carbon dioxide detected, Off means no carbon dioxide (clear)
```

Comment:
```suggestion
    # On means abnormally high levels of carbon dioxide detected, Off means normal levels
```

Reply:
CO2 dropped - so closing this conversation.